spec/app/domain/authentication/authn_k8s/application_identity_spec.rb

```diff
@@ -136,7 +136,7 @@
 
     context "Application identity in host id" do
       let(:host_annotations) { [] }
-      let(:host_id) { "#{namespace}/#{k8s_resource_name}/#{k8s_resource_value}" }
+      let(:host_id) { "accountName:host:#{namespace}/#{k8s_resource_name}/#{k8s_resource_value}" }
```

Comment:
I suggest we factor this out like we have for namespace, resource_name, and resource_value for the sake of uniformity